bot/bot_interaction.py

```diff
@@ -0,0 +1,27 @@
+from typing import Final
+
+import telebot
+from telebot import types
+
+from constants import BOT_TOKEN
+
+bot = telebot.TeleBot(BOT_TOKEN, parse_mode=None)
+BOT_USERNAME: Final = '@game_whisper_bot'
+
+
+@bot.message_handler(commands=['start', 'help'])
+def send_welcome(message):
+    bot.reply_to(message, ' Welcome to Gaming Whisper')
+    markup = types.ReplyKeyboardMarkup(row_width=2)
+    button_1 = types.InlineKeyboardButton('Game Profile')
```

Comment:
Give an appropriate name for this button here (something like game_profile_button), because if it's gonna be required later in the code, you would type _button_1_ and collaborators would have no idea what kind of button this is exactly. Same applies to the button_2. That's a bad naming